backend/src/controllers/game_controller.spec.ts

```diff
@@ -0,0 +1,134 @@
+    expect(res.body[0].title).toBe(game.title);
+    expect(res.body[0].questionsNumber).toBe(game.questionsNumber);
+    expect(res.body[0].isPublic).toBe(game.isPublic);
+    expect(res.body[0].mode).toBe(game.mode);
+    expect(res.body[0].idSpotifyPlaylist).toBe(game.idSpotifyPlaylist);
+  });
+});
+
+describe("Test GET, DELETE, GET game", () => {
+  it("should return pre-created game", async () => {
+    const res = await request(server)
+      .get(`/games/${game.id}`)
+      .send();
+    expect(res.status).toEqual(200);
+    expect(res.body.id).toBe(game.id);
+    expect(res.body.startDate).toBe("2020-01-21T15:24:16.331Z");
+    expect(res.body.isEnded).toBe(game.isEnded);
+    expect(res.body.score).toBe(game.score);
+    expect(res.body.title).toBe(game.title);
+    expect(res.body.questionsNumber).toBe(game.questionsNumber);
+    expect(res.body.isPublic).toBe(game.isPublic);
+    expect(res.body.mode).toBe(game.mode);
+    expect(res.body.idSpotifyPlaylist).toBe(game.idSpotifyPlaylist);
+  }),
+    it("should delete pre-created game", async () => {
+      const res = await request(server)
+        .delete(`/games/${game.id}`)
+        .send();
+      expect(res.status).toEqual(204);
+    }),
+    it("should not return pre-created game", async () => {
+      const res = await request(server)
+        .get(`/games/${game.id}`)
+        .send();
+      expect(res.status).toEqual(404);
+    });
+});
+
+describe("Test POST, PUT, GET", () => {
+  const game: Game = new Game();
+  game.startDate = new Date(1579622125436);
+  game.isEnded = false;
+  game.score = 0;
+  game.title = "new title";
+  game.questionsNumber = 10;
+  game.isPublic = false;
+  game.mode = 1;
+  game.idSpotifyPlaylist = "new_id";
+
+  it("should create a new game", async () => {
+    const res = await request(server)
+      .post("/games")
+      .send({
+        startDate: 1579622125436,
+        isEnded: game.isEnded,
+        score: game.score,
+        title: game.title,
+        questionsNumber: game.questionsNumber,
+        isPublic: game.isPublic,
+        mode: game.mode,
+        idSpotifyPlaylist: game.idSpotifyPlaylist,
+      });
+    expect(res.status).toEqual(200);
+    game.id = res.body.id;
+  }),
+    it("should update the game", async () => {
+      const res = await request(server)
+        .put(`/games/${game.id}`)
+        .send({
+          questionsNumber: 54,
+          title: "AnOtHeR nEw TiTlE",
+        });
+      expect(res.status).toEqual(200);
+    }),
+    it("should get the updated game", async () => {
+      const res = await request(server)
+        .put(`/games/${game.id}`)
+        .send();
+      expect(res.status).toEqual(200);
+      expect(res.body.id).toBe(game.id);
+      expect(res.body.startDate).toBe("2020-01-21T15:55:25.436Z");
+      expect(res.body.isEnded).toBe(game.isEnded);
+      expect(res.body.score).toBe(game.score);
+      expect(res.body.title).toBe("AnOtHeR nEw TiTlE");
```

Comment:
How about leet speak here?

Reply:
haha I'll think about it for other tests :p